src/StoreBase.ts

```diff
@@ -71,14 +72,10 @@ export abstract class StoreBase {
 
     // If you trigger a specific set of keys, then it will only trigger that specific set of callbacks (and subscriptions marked
     // as "All" keyed).  If the key is all, it will trigger all callbacks.
-    protected trigger(keyOrKeys?: string|number|(string|number)[]) {
-        let throttleMs: number | undefined;
-        if (this._throttleMs !== undefined) {
-            throttleMs = this._throttleMs;
-        } else {
-            // If the store doens't define any throttling, pick up the default
-            throttleMs = Options.defaultThrottleMs;
-        }
+    protected trigger(keyOrKeys?: KeyOrKeys) {
+        const throttleMs: number | undefined = this._throttleMs !== undefined
```

Comment:
The type on the LHS is un-needed, since the RHS will type it to number (not number|undefined which seems to be an old typo)